### PR TITLE
Adds readme with debugger.html version information to mc.

### PR DESCRIPTION
**Pull request by yurydelendik**
Associated Issue: will help with bisecting

### CODE REVIEW

File: bin/writeReadme.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  } catch (ex) {
+    return { name, version: "n/a" };
+  }
+}
+
+const packageOfInterest = [
+  "babel-plugin-transform-es2015-modules-commonjs",
+  "babel-preset-react",
+  "react",
+  "react-dom",
```
Comment (jasonLaster) [inline bin/writeReadme.js]: we exclude some of these from the bundle: 

https://github.com/ahuth/debugger.html/blob/fix-warnings/webpack.config.js#L53-L61

File: bin/writeReadme.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    proc.on("close", code => resolve(version.trim()));
+    proc.on("error", error => {
+      if (error.code == "ENOENT") {
+        reject(new Error("Could not find git."));
+        return;
+      }
+      reject(error);
+    });
+  });
+}
```
Comment (jasonLaster) [inline bin/writeReadme.js]: :thumbsup:

File: bin/writeReadme.js (hunk 1 above)
Comment (yurydelendik) [reply bin/writeReadme.js]: right, we might need to know difference anyway

Comment (yurydelendik) [reply bin/writeReadme.js]: list is incomplete too

**Pull request merged**